### ISSUE TITLE
Fetching submodules in parallel 

### ISSUE THREAD

**Issue by jabranham** [magit/magit#3261]
Currently, submodule fetching with (from magit-status) `f m` is done one-at-a-time. git 2.8+ supports doing this in parallel, though, with: `git fetch --recurse-submodules -j7`, which cuts down the amount of time this takes considerably. Is it possible to teach `magit-submodule-fetch` this behavior?

**Comment by tarsius** [comment]
Yes, I will look into that.

**Comment by tarsius** [comment]
Done. See the commit for why that wasn't already done before.
